tasks/fh-build.js

```diff
@@ -43,7 +42,7 @@ module.exports = function(grunt) {
    * If grunt config contains an array property called 'fhignore',
    * its elements will be excluded from the tarball.
    */
-  var mergePatterns = function(bundle_deps) {
+  var mergePatterns = function (bundle_deps) {
```

Comment:
-1 on this change. JS Beautify defaults does not include the space, only seems to be IDEA/WebStorm that adds it with its default formatting. Would introduce a lot of churn!